client/packages_test.go

```diff
@@ -87,3 +89,45 @@ func (cs *clientSuite) TestClientPackages(c *check.C) {
 		},
 	})
 }
+
+const (
+	appID = "chatroom.ogra"
+)
+
+func (cs *clientSuite) TestClientPackageCallsEndpoint(c *check.C) {
+	_, _ = cs.cli.Package(appID)
```

Comment:
This is a habit I picked up from years of Ruby testing with RSpec. The style is for much more focussed tests with ideally a single assertion per test.

If what you're suggesting is considered idiomatic Go then I'm happy to change it.